config/crd/bases/operator.kyma-project.io_watchers.yaml

```diff
@@ -130,6 +145,7 @@ spec:
             - field
             - gateway
             - labelsToWatch
+            - resourceToWatch
```

Comment:
I didn't. It was added automatically because the `v1alpha1.Watcher` type uses `v1beta1.WatcherSpec`. If we want to have separate specs between `v1alpha1` and `v1beta1` we can't avoid duplication starting from this code state. I think it's fine to keep it as is since `v1alpha1` is [deprecated](https://github.com/kyma-project/lifecycle-manager/blob/c044a5b5c9d961bf5bfb680c70cfc9305c0c5c4c/api/v1alpha1/watcher_types.go#L27).